### PR TITLE
TechArea notes for Aug 23 2022

### PR DESCRIPTION
**Pull request by edquist**
@brianhlin - my first time taking notes; did it on the fly from an empty text file.  Adapted here to the standard TA notes format.  Sorry for missing some bits.  Next time I will start with the template which should hopefully better guide the note taking.

### CODE REVIEW

File: docs/meetings/2022/TechArea20220823.md [hunk 1]
```diff
@@ -2,38 +2,34 @@
...
 
-| # of tickets | &Delta; | State             |
-|--------------|---------|-------------------|
-| 187          | -7      | Open              |
-| 38           | +6      | Selected for Dev  |
-| 35           | +5      | In Progress       |
-| 18           | +1      | Dev Complete      |
-| 3            | +0      | Ready for Testing |
-| 0            | +0      | Ready for Release |
+Lots of tickets should be backlogged.
```
Comment (brianhlin) [inline docs/meetings/2022/TechArea20220823.md:16]: Could you revert the changes here? I still need to update these numbers and need to include them

File: docs/meetings/2022/TechArea20220823.md [hunk 2]
```diff
@@ -2,38 +2,34 @@
...
 
 ## OSG Software Team
 
--  Use COManage IDs in Topology wherever possible!
--  AI (Mat): build stash-origin chart (SOFTWARE-5258)
--  AI (BrianL): set up expiration notification template and workflow; contact COManage developers for assistance
-    if needed
--  AI (Carl): complete custom workflows ticket; meet with BrianL and Mat to discuss use cases for workflows
+-  k8s hackathon today, already discussed things on the docket
```
Comment (brianhlin) [inline docs/meetings/2022/TechArea20220823.md:20]:
```suggestion
-  Kubernetes hackathon today, already discussed things on the docket
```

These notes go to a wider audience so I try to be a bit more explicit

File: docs/meetings/2022/TechArea20220823.md [hunk 3]
```diff
@@ -2,38 +2,34 @@
...
+-  fields (like github username) are editible in comanage;
```
Comment (brianhlin) [inline docs/meetings/2022/TechArea20220823.md:21]:
```suggestion
-  Fields (like github username) are editible in comanage;
```

File: docs/meetings/2022/TechArea20220823.md [hunk 4]
```diff
@@ -2,38 +2,34 @@
...
+   would be good to test editing especially if we are non-admins
+-  Marco: has PR for GWMS item; troubleshooting Mats's issue
+-  AI (BrianL, Mat, Carl) discuss all the container building use cases
+-  AI (Mat): check in with Fabio to see where PR for hawaii origin stands
+
 
 ### Discussion
 
+-   Do we want to fork off existing slate chart, or do we just make the changes?
```
Comment (brianhlin) [inline docs/meetings/2022/TechArea20220823.md:30]:
```suggestion
-   Do we want to fork off existing SLATE chart, or do we just make the changes?
```

File: docs/meetings/2022/TechArea20220823.md [hunk 5]
```diff
@@ -45,19 +41,19 @@
 
 ### Support Update
 
--   FIU (Mat) working with FIU to resolve connection issues while attempting to mirror OSG Software repos.
-
--   FNAL (Carl) worked with Yuijun and Kevin Retzke on Gratia probe changes. 
-    Software team work is done for now;
-    at this point Yuijun and Kevin should discuss their requirements.
+...
```
Comment (brianhlin) [inline docs/meetings/2022/TechArea20220823.md:44]:
```suggestion
None this week
```

File: docs/meetings/2022/TechArea20220823.md [hunk 6]
```diff
@@ -45,19 +41,19 @@
...
+    -   gratia-probe
+    -   stashcp
 
 ### Discussion
 
-    - Release xrootd-monitoring-shoveler and HTCondor 9.0.16 this week.
+-   TimT: note prce2 more strict about character classes in regex (hyphen must go at the end `[abc-]`)
+-   BrianL: need to find victims to update from stable series to 9.12
+-   AI (TimT): reach out to John Thiltges about feasibility of updating to 10.0?
+    (TimT: in a couple weeks)
```
Comment (brianhlin) [inline docs/meetings/2022/TechArea20220823.md:59]:
```suggestion
    (TimT: 9.12.0 in a couple weeks)
```

**Review by brianhlin — APPROVED** [review]
Looks mostly good! Pre-approving

File: docs/meetings/2022/TechArea20220823.md (hunk 1 above)
Comment (edquist) [reply docs/meetings/2022/TechArea20220823.md]: Done

File: docs/meetings/2022/TechArea20220823.md (hunk 6 above)
Comment (edquist) [reply docs/meetings/2022/TechArea20220823.md:72]: Ah i guess i misunderstood and thought he was referring to 10.0.

**Comment by edquist** [comment]
@brianhlin, looks like the link checker failed (https://osg-htc.org/docs/security/user-certs/).  i don't have admin override so i'll leave it to you if you'd like to merge.
